RSSReader-Maruyama/DataManagers/UserDataManager.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  LoginChecker.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class UserDataManager {
+    
+    private let USER_DATA_KEY = "userData"
+    lazy var userDefaults = UserDefaults.standard
+
+    
+    func exists() -> Bool {
+        let user = userDefaults.data(forKey: USER_DATA_KEY)
+        
+        return user != nil
+    }
+    
+    func load() -> User? {
+        guard let userData = userDefaults.data(forKey: USER_DATA_KEY) else {
+            return nil
+        }
+        let user = try? JSONDecoder().decode(User.self, from: userData)
```

Comment:
```suggestion
        return try? JSONDecoder().decode(User.self, from: userData)
```
ここで変数に一度格納する必要性は無いと思います。そのままreturnしましょう。